src/ctx.h

```diff
@@ -70,6 +70,8 @@ class AddressInfo {
     // Return the address space that this address resides in.
     unsigned getAddressSpace() const { return getType()->getAddressSpace(); }
 
+    static llvm::Type *GetAddressElementType(const PointerType* pt);
```

Comment:
Function name is not descriptive and there are no comments. What about `GetPointeeLLVMType`?